client/src/document.js

```diff
@@ -2,6 +2,9 @@ import React from "react";
 import { Link } from "@reach/router";
 
 import { NoMatch } from "./routing";
+
+// Ingredients
```

Comment:
Isn't that obvious? :)

Reply:
:joy: Sorry it's a habit to name a group when grouping imports together